### DISCUSSION TITLE
How to do a sum query

### DISCUSSION THREAD

**Discussion by patcito** [supabase/supabase#432] [Questions]
What would be the recommended way to do a sum query such as `select sum(amount) from deposits`?

Thanks,

-Patrick

**Comment by steve-chavez** [comment]
Hey @patcito,

You can create a view for that:

```sql
CREATE VIEW deposits_sum AS
SELECT sum(amount)
FROM deposits
```

Then you can call the view through `supabase-js`, same as any other table.
